.github/workflows/ci.yaml

```diff
@@ -70,13 +70,8 @@ jobs:
       - name: Run unit tests
         run: go run mage.go coverage
 
-      - name: Run e2e tests against the example
-        shell: bash
-        run: >
-          for image in $ENVOY_IMAGES; do
-              echo "Running e2e with Envoy image $image"
-              ENVOY_IMAGE=$image go run mage.go e2e
-          done
+      - name: Run e2e tests
+        run: go run mage.go e2e
```

Comment:
Somehow we did not realize now we are only testing against the default image :( cc @M4tteoP

Reply:
I don't get why in the first place I changed it, the entry point to e2e tests is still the same mage command 🤔. Sorry about that, running the CI with this change reverted here: https://github.com/corazawaf/coraza-proxy-wasm/pull/227